### DISCUSSION TITLE
Network graph in repo Insights on Firefox 97.0.1 for macOS is broken

### DISCUSSION THREAD

**Discussion by unixorn** [community/community#12150] [Other Feature Feedback, Questions, & Ideas]
Here's what it looks like in Firefox 97.0.1 on macOS 12.2.1:

<img width="1080" alt="Screen Shot 2022-02-26 at 9 16 51 AM" src="https://user-images.githubusercontent.com/23920/155850599-dd48713e-7998-4bd5-ae32-72dc91372a24.png">

And here it is on Safari on the same MacBook Pro:

<img width="817" alt="working-network-graph" src="https://user-images.githubusercontent.com/23920/155850673-a1146227-4783-40e8-b162-015446409e1d.png">

These are both views of https://github.com/unixorn/zsh-quickstart-kit/network

I can drag the graph around in the tiny little window and see more of it that way, but I'm looking at it through a keyhole.

**Comment by Tsoccerguy3** [comment]
I'm experiencing this as well in Firefox 97.0.1 on Window 10x64 and i can not even see 1 cm of the graph

**Comment by ghost** [comment]
I also have the same issue. I try both Firefox and Chrome on desktop and mobile that I can't see the graph

![firefox_uVHI6gb6QR](https://user-images.githubusercontent.com/68732241/155879942-1e1ec22a-45e3-49c3-acd6-cc165252398e.png)

**Comment by akinomyoga** [comment]
This is a duplicate of #12061.
